docs/i18n/ja-jp/docusaurus-plugin-content-docs/current/concepts/shortcuts.md

```diff
@@ -68,64 +68,57 @@ app.shortcut('open_modal', async ({ shortcut, ack, context, logger }) => {
-  // callback_id が 'open_modal' と一致し type が 'message_action' と一致する場合のみミドルウェアが呼び出される
-  app.shortcut({ callback_id: 'open_modal', type: 'message_action' }, async ({ shortcut, ack, context, client, logger }) => {
-    try {
-      // ショートカットリクエストの確認
-      await ack();
+```javascript
+// callback_id が 'open_modal' と一致し type が 'message_action' と一致する場合のみミドルウェアが呼び出される
+app.shortcut({ callback_id: 'open_modal', type: 'message_action' }, async ({ shortcut, ack, context, client, logger }) => {
+  try {
+    // ショートカットリクエストの確認
+    await ack();
 
-      // 組み込みの WebClient を使って views.open API メソッドを呼び出す
-      const result = await app.client.views.open({
-        // `context` オブジェクトに保持されたトークンを使用
-        token: context.botToken,
-        trigger_id: shortcut.trigger_id,
-        view: {
-          type: "modal",
-          title: {
-            type: "plain_text",
-            text: "My App"
-          },
-          close: {
-            type: "plain_text",
-            text: "Close"
+    // 組み込みの WebClient を使って views.open API メソッドを呼び出す
+    const result = await app.client.views.open({
+      // `context` オブジェクトに保持されたトークンを使用
+      token: context.botToken,
+      trigger_id: shortcut.trigger_id,
+      view: {
+        type: "modal",
+        title: {
+          type: "plain_text",
+          text: "My App"
+        },
+        close: {
+          type: "plain_text",
+          text: "Close"
+        },
+        blocks: [
+          {
+            type: "section",
+            text: {
+              type: "mrkdwn",
+              text: "About the simplest modal you could conceive of :smile:\n\nMaybe <https://api.slack.com/reference/block-kit/interactive-components|*make the modal interactive*> or <https://api.slack.com/surfaces/modals/using#modifying|*learn more advanced modal use cases*>."
+            }
           },
-          blocks: [
-            {
-              type: "section",
-              text: {
+          {
+            type: "context",
+            elements: [
+              {
                 type: "mrkdwn",
-                text: "About the simplest modal you could conceive of :smile:\n\nMaybe <https://api.slack.com/reference/block-kit/interactive-components|*make the modal interactive*> or <https://api.slack.com/surfaces/modals/using#modifying|*learn more advanced modal use cases*>."
+                text: "Psssst this modal was designed using <https://api.slack.com/tools/block-kit-builder|*Block Kit Builder*>"
               }
-            },
-            {
-              type: "context",
-              elements: [
-                {
-                  type: "mrkdwn",
-                  text: "Psssst this modal was designed using <https://api.slack.com/tools/block-kit-builder|*Block Kit Builder*>"
-                }
-              ]
-            }
-          ]
-        }
-      });
-
-      logger.info(result);
-    }
-    catch (error) {
-      logger.error(error);
-    }
-  });
-  ```
+            ]
+          }
+        ]
```

Comment:
this is just a code snippet pulled from a <details> element no code was actually changed promise